website/content/docs/k8s/installation/install.mdx

```diff
@@ -199,6 +200,43 @@ NAME: consul
 ...
 ```
 
+#### Enable Consul Service Mesh on select namespaces
+
+By default, Consul Service Mesh is enabled on all namespaces within a Kubernetes cluster. You can restrict this to a subset of namespaces by specifying a `namespaceSelector` that matches a label attached to each namespace denoting whether to enable Consul service mesh.
+<CodeBlockConfig filename="config.yaml">
+
+```yaml
+global:
+  name: consul
+connectInject:
+  enabled: true
+  default: true
```

Comment:
Should we call this out a bit more that setting `default` to `true` is required for this to work?

Reply:
Thanks added!